### ISSUE TITLE
Mod Conflct - Cause Unclear

### ISSUE THREAD

**Issue by MaheEvil** [isXander/Zoomify#83]
Heyo, I am a mod developer and recently I have been getting few user issue reports of getting dependency errors with cloth-config even tho the correct version of cloth config mod was in the mods folder. ~~After some investigation it was narrowed down to JIJ inclusion of cloth-config in this mod causing the issue.~~

### ~~Please consider one of two solutions listed bellow~~
~~- **Stop including the cloth-config jar with the mod and have the user manually install it** - While this is inconvenient for users, This is the solution I would be most happier with as this would eliminate this problem all-together, And this would be more beneficial to Cloth-config developers.~~   
~~- **At the very least include the latest CF stable release version for given the Minecraft version** - I am not exactly thrilled to deal with users complaining about dependency issues through no fault of either me or the user, wasting both parties time. Nor do I want the solution to be having them remove your mod. *Yes I do plan on changing my fabric.mod.json dependency configuration to allow older versions of cloth-config, which is irrelevant and I should not have to do that*~~

Retracting that, half-ly, context below

**Comment by isXander** [comment]
I don't JiJ cloth at all.

**Comment by MaheEvil** [comment]
Heyo mate, there is a slight misunderstanding with settxi-cloth-cofig and cloth-config in that case, sorry for my aggressive attitude before (just had too much reports with this I had to deal with, and I just quickly checked the build.gradle files).

This leaves me wondering as to why this dependency error was caused with zoomify, or is it faulty user-reports?

Thanks and sorry about the aggressive tonality the issue was written in

**Comment by isXander** [comment]
Could you send the issue so I can take a look?

**Comment by MaheEvil** [comment]
> Could you send the issue so I can take a look?

Yea, both of them got Incompatible modset error like this.
![image](https://user-images.githubusercontent.com/66691814/186441484-b30a6088-5d29-4214-ab7a-fae07d4e812d.png)
and I have confirmed their modlist had the correct version of cloth-config.

however I did try to reproduce this error on my own a bit ago and was unable to do so, but I got two user reports.

Here is that specific user's modlist
![image](https://user-images.githubusercontent.com/66691814/186441845-f61ba6eb-34b6-45c9-a4e2-3917b6c6f8cb.png)
![image](https://user-images.githubusercontent.com/66691814/186441864-baab6fe0-47fe-4e79-bbe4-d977aa92f394.png)
